### ISSUE TITLE
Using example template created date shows at top rather than bottom

### ISSUE THREAD

**Issue by boardtc** [akosbalasko/yarle#360]
I converted 34 notebook from evernote to Obsidian, your tool is so great, thanks!!

I used the suggested template, just deleting the location:

`
{title-block}# {title}{end-title-block}

{tags-block}
---
Tag(s): {tags}

---
{end-tags-block}

{content-block}{content}{end-content-block}

    {created-at-block}    Created at: {created-at}{end-created-at-block}
    {updated-at-block}    Updated at: {updated-at}{end-updated-at-block}
    {source-url-block}    Source: {source-url}{end-source-url-block}
    {notebook-block}      Notebook: {notebook}{end-notebook-block}
    {link-to-original-block}      Link to original content: {link-to-original}{end-link-to-original-block}
`

However the created at is showing at the top rather than the bottom:
![This is an image](https://snipboard.io/MwAJXN.jpg)

**Comment by boardtc** [comment]
fyi, the life preview in Obsidian shows like that as horizontal lines at the top of a note are treated as YAML, not lines.

**Comment by boardtc** [comment]
You probably have a backlog of work...my move away for evernote on hold for this one....

**Comment by akosbalasko** [comment]
@boardtc I tested it locally and it looks working perfectly for me. Didn't you miss clicking on the Save button when you updated the template?

**Comment by boardtc** [comment]
Sorry for the delay! The other bug I reported held me up a bit but I reinstalled today in order to rerun Yarle. I picked one of my evernote backed up notebooks, copied the template above and saved and it's fine! 
Can I suggest that you remove the save button and just autosave or else put up a warning message when moving to step 5 if it is not saved.
Thanks!
I hope to make time soon then to redo the conversion and retest the output.
